Analysis/bin/ShuffleMergeSpectral.cxx

```diff
@@ -48,6 +46,7 @@ struct Arguments {
     run::Argument<double> start_entry{"start-entry", "starting ratio from which file will be processed", 0};
     run::Argument<double> end_entry{"end-entry", "end ratio until which file will be processed", 1};
     run::Argument<double> exp_disbalance{"exp-disbalance", "maximal expected disbalance between low pt and high pt regions",0};
+    run::Argument<std::string> compression_algo{"compression-algo","ZLIB, LZMA, LZ4","LZ4"};
```

Comment:
put as default LZMA with lvl=9. We will use the first big shuffle&merge production to test if decompression is a bottleneck for the training. If yes, we can consider another compression, if no - better to have smaller files.